main.py

```diff
@@ -6,9 +6,9 @@
 import glueops.setup_logging
 import os
 
-log_level = getattr(glueops.setup_logging,
+log_level = getattr(glueops.setup_logging.logging,
```

Comment:
you probably don't need getattr for these, since we're building on top of native logging which accepts the string representations of log levels